openmetadata-ui/src/main/resources/ui/src/components/AppRouter/AuthenticatedAppRouter.tsx

```diff
@@ -497,6 +498,12 @@ const AuthenticatedAppRouter: FunctionComponent = () => {
       />
 
       <Route component={ClassificationRouter} path="/tags" />
+      <Route
+        exact
+        component={TagPage}
+        // hasPermission={tagCategoryPermission}
```

Comment:
Can we update this with actual permission

Reply:
Sure. This is still left. @karanh37 will guide me for this.